config/config.staging.ts

```diff
@@ -0,0 +1,22 @@
+import { Environment } from './types';
+
+export default {
+  env: Environment.STAGING,
+  web: {
+    host: process.env.HOST || '0.0.0.0',
+    port: process.env.PORT || 4002,
+    routes: {
+      cors: {
+        origin: ['https://twine-visitor-staging.herokuapp.com', 'https://admin.twine-together.com'],
```

Comment:
I don't think we should add `'https://admin.twine-together.com'`. It won't work with our cookies, and there is no staging version of the app to point to this server. (Unless this is being added for testing)